src/instrument/startup.py

```diff
@@ -20,6 +20,7 @@
 from .devices import *  # noqa: F403
```

Comment:
could remove

Reply:
Only if guarnieri is going to be the default way of creating devices right? Otherwise, they will both need to co-exist somehow. Are there examples with guarnieri already?

Reply:
Discusssed .. leaving it in.